backend/api/src/place-bet.ts

```diff
@@ -432,13 +425,15 @@ export const processNewBetResult = async (
         )
       }
     } else {
-      fbTrans.update(
-        contractDoc,
+      await updateContract(
+        pgTrans,
+        contract.id,
         removeUndefinedProps({
           pool: newPool,
           p: newP,
           totalLiquidity: newTotalLiquidity,
-        })
+        }),
+        { silent: true }
```

Comment:
Hmm, shouldn't this not be silent so that the pool is correct for traders?

Reply:
currently we pass the contract through the broadcastBet function and broadcast a contract update alongside the bet update. very confusing